content/blog/2022/02/camel315-whatsnew/index.md

```diff
@@ -0,0 +1,130 @@
+- `--console` - Includes the new Camel developer console (Very early prototype, plain text mode only)
+
+We also made it easy to run using wildcards, for example to run with all files included from current dir:
+
+     camel run *
+
+You can also run directly from GitHub via:
+
+     camel run https://github.com/apache/camel-examples/tree/main/examples/jbang
+
+The `--reload` option has been improved, and in the next release Camel can also do updates to Java classes.
+However, in this release the reloading is only Camel route source files.
+
+See more details at the [Camel JBang documentation](/manual/camel-jbang/)
+
+### Developer Console
+
+This release comes with an early prototype of the new Camel Developer Console.
+The developer console is intended for development purposes, by giving relevant
+information to the developer about the running Camel application.
+
+The console is based on a pluggable architecture (similar to health-checks), which
+makes it possible for individual components to provide their own consoles, and also
+to make 3rd-party consoles as well.
+
+Camel will come with a set of out-of-the-box consoles, such as general information
+about the Camel version, which components are in use, real-time metrics of the routed
+messages, and details of the last N throw exceptions etc.
+
+We will continue to work on this functionality in the upcoming releases.
+
+We also plan to integrate with [Quarkus Dev UI](https://quarkus.io/guides/dev-ui),
+so you will have a lot of Camel consoles when using Camel on Quarkus.
+
+### Quarkus
+
+The upcoming Camel Quarkus 2.8 release will be upgraded to Camel 3.15. 
+
+### Spring Boot
+
+We have upgraded to the latest Spring Boot 2.6 release.
+
+### Better Java 17 support
+
+Although Java 17 is not officially supported, we did improve a few Camel components to make them work with Java 17.
+The official support is Java 11.
+
+### New components
+
+This release has a number of new components:
+
+- `camel-dynamic-router` - The Dynamic Router component routes exchanges to recipients, and the recipients (and their rules) may change at runtime.
+- `camel-huaweicloud-frs` - Face Recognition Service (FRS) is an intelligent service that uses computers to process, analyze, and understand facial images based on human facial features.
+- `camel-knative` - Send and receive events from Knative.
+- `camel-milo` - Connect to OPC UA servers using the binary protocol for browsing the node tree.
+- `camel-cloudevents` - Camel support for the CloudEvents specification
+- `camel-jsh-dsl` - Camel DSL with JavaShell
+
+## Upgrading
+
+Make sure to read the [upgrade guide](/manual/camel-3x-upgrade-guide-3_15.html) if you are upgrading from a previous Camel version.
+
+## Release Notes
+
+You can find more information about this release in the with a list of JIRA tickets resolved in the release: 
+
+- [release notes 3.15](/releases/release-3.15.0/)
```

Comment:
We need a `release-3.15.0.md` [here](https://github.com/apache/camel-website/tree/main/content/releases)